controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -44,6 +44,7 @@ const (
 	StorageConsumerAnnotation     = "ocs.openshift.io.storageconsumer"
 	StorageRequestAnnotation      = "ocs.openshift.io.storagerequest"
 	StorageCephUserTypeAnnotation = "ocs.openshift.io.cephusertype"
+	StorageProfileLabel           = "ocs.openshift.io/storageprofile"
```

Comment:
I don't see this being used anywhere in the PR.

Reply:
I believe we agreed to have this as an annotation not as a label.